### ISSUE TITLE
Support for create_bucket argument.

### ISSUE THREAD

**Issue by zcherry17** [terraform-google-modules/terraform-google-scheduled-function#80]
### TL;DR

The [event-function module](https://registry.terraform.io/modules/terraform-google-modules/event-function/google/latest?tab=inputs) has a `create_bucket` input argument that should also be supported by this module.
The inclusion of this input argument permits the use case of using a bucket that already exists for the storage of function deployment sources when `create_bucket` is false and `bucket_name` points to an existing bucket.

### Terraform Resources

```markdown
https://registry.terraform.io/modules/terraform-google-modules/event-function/google/latest?tab=inputs
```

### Detailed design

```markdown
In main.tf:

module "main" {
    ...
    create_bucket = var.function_create_bucket
    ...
}

In variables.tf:

variable "function_create_bucket" {
    type = bool
    default = true
    description = "Whether to create a new bucket or use an existing one. If false, `bucket_name` should reference the name of the alternate bucket to use."
}
```

### Additional information

I am more than happy to create a PR for this, and I think this should be supported since finer control is always nice, especially since that finer control is already supported by the underlying event functions module this module makes use of.

**Comment by bharathkkb** [comment]
Happy to review a PR for this!

**Comment by DarkWanderer** [comment]
Does something like #147 work?
